internal/parsers/db/put.go

```diff
@@ -36,12 +36,14 @@ func (db *CsvDB) Put(data []Data) error {
 		// Calculate a unique index for each key
 		keyString := key.CountryCode().String() + key.LocationCode().String()
 
-		if _, exists := uniqueKeys[keyString]; exists {
-			continue // Skip duplicates
+		if index, exists := uniqueKeys[keyString]; exists {
```

Comment:
can you add a comment about why we overwrite only these fields? can be tricky for a next reader